src/Spryk/Model/Spryk/Definition/Argument/Callback/ZedDependencyModelInterfaceTargetFilenameCallback.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Spryk\Model\Spryk\Definition\Argument\Callback;
+
+use SprykerSdk\Spryk\Model\Spryk\Definition\Argument\Collection\ArgumentCollectionInterface;
+
+class ZedDependencyModelInterfaceTargetFilenameCallback implements CallbackInterface
+{
+    /**
+     * @return string
+     */
+    public function getName(): string
+    {
+        return 'ZedDependencyModelInterfaceTargetFilenameCallback';
```

Comment:
Why is this one needed?

Reply:
I think i can replace it with just a value in targetFilename.
And if you ask why i created this one, then because the previous one belongs to another namespace, so i don't wanna couple them.

Reply:
deleted